harness/determined/profiler.py

```diff
@@ -762,14 +773,14 @@ def convert_to_post_format(self) -> List[TrialProfilerMetricsBatch]:
 class SimpleCpuUtilCollector:
     def measure(self, batch_idx: int) -> Measurement:
         cpu_util = psutil.cpu_percent()
-        timestamp = datetime.utcnow()
+        timestamp = datetime.now(timezone.utc)
```

Comment:
perhaps the API should take just take unix timestamp nanos instead?

Reply:
I don't have an opinion either way. Let's discuss at standup.